examples/room/lod/lodmanager/user_entity.go

```diff
@@ -0,0 +1,33 @@
+// © 2019-2025 Diarkis Inc. All rights reserved.
+
+package lodmanager
+
+import (
+	"fmt"
+	"sync"
+	"time"
+)
+
+// UserEntity represents a user's position and payload data
+type UserEntity struct {
+	X                int32
+	Y                int32
+	Payload          []byte
+	Remember         map[string]time.Time
+	RememberMutex    sync.RWMutex
+	ChangedAfterSend bool
```

Comment:
QUESTION: Why public fields?
```suggestion
	X                int32
	Y                int32
	Payload          []byte
	m                map[string]time.Time // Remember map
	mu               sync.RWMutex
	ChangedAfterSend bool
```

Reply:
It was just a remnant of the print debugging...